evopy/evopy.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, fitness_function, individual_length, warm_start=None, generat
         self.maximize = maximize
         self.strategy = strategy
         self.random_seed = random_seed
```

Comment:
Maybe this assignment is no longer necessary then? I think all references to `self.random_seed` have been replaced with a `self.random` reference, so this one can be purely a constructor parameter

Reply:
Definitely a good point, I thought about this, but perhaps it's nice for tractability  and debugging: "Which seed is used for this class". As it's actually not trivial to find the seed of a `self.random` without some dirty hacks :(

Reply:
Valid point, that does sound like it might become useful some time.